Automattic/wp-calypso
# Gutenberg: Update Publicize extension to use @wordpress packages

This PR updates the Publicize extension to use @wordpress packages instead of the global `wp` variable (where possible). This follows the example of #26549, which did the same for the Tiled Gallery block.

To achieve this, we also introduce a couple of additional @wordpress packages that weren't added as dependencies (cc @vindl):
* @wordpress/compose
* @wordpress/plugins

This also reduces the build size by a bit:

Before:
![](https://cldup.com/s05b82V4As.png)

After:
![](https://cldup.com/XagPuoQBIr.png)

To test:
* Checkout this branch.
* Run a Jetpack site and checkout https://github.com/Automattic/jetpack/pull/9963/files on it.
* Run `npm run sdk:gutenberg -- --editor-script=client/gutenberg/extensions/publicize/publicize-gutenberg-plugin.jsx --output-dir=/DIR_TO_JETPACK/jetpack/modules/publicize/assets/ --output-editor-file=block`
* Verify the Publicize extension still appears properly in the wp-admin (you'll notice the Jetpack icon in the regular plugin sidebar and the post edit header settings area).
